catalog/transformer/catalog-transformer-tika-input/src/main/java/ddf/catalog/transformer/input/tika/TikaInputTransformer.java

```diff
@@ -223,6 +263,9 @@ public Metacard transform(InputStream input, String id)
             }
 
             String metacardContentType = metacard.getContentTypeName();
+            metacard.setAttribute(new AttributeImpl(Core.DATATYPE,
+                    getDatatype(metacardContentType)));
```

Comment:
Just noticed that there should probably be a check on metacardContentType before setting the dataset here 
```
If(!StringUtils.isNotBlank(metacardContentType)) {
...
}
```